packages/terra-layout/src/LayoutSlidePanel.module.scss

```diff
@@ -26,10 +26,10 @@
   }
 
   .panel {
-    background-color: var(--layout-slide-panel-background-color, #fff);
+    background-color: var(--terra-layout-slide-panel-background-color, #fff);
     position: absolute;
     transform: translate3d(-100%, 0, 0);
-    width: var(--layout-slide-panel-desktop-panel-width, 18.571rem);
+    width: var(--terra-layout-slide-panel-desktop-panel-width, 18.571rem);
```

Comment:
I think including `desktop` here is odd. I also noticed on `background-color` we use `var(--terra-layout-slide-panel-background-color, #fff);` so theres some inconsistency here.

I'm thinking through this and I find if we follow how we've named other items under a class, `--terra-layout-slide-panel-panel-width` is a bit odd as well. 

What are peoples thoughts on changing the class to `.pane` and going with the following?
```
background-color: var(--terra-layout-slide-panel-pane-background-color, #fff);
width: var(--terra-layout-slide-panel-pane-width, 18.571rem);
```

Reply:
Edit, let's just keep the class name as `.panel` and go with 
```
background-color: var(--terra-layout-slide-panel-panel-background-color, #fff);
width: var(--terra-layout-slide-panel-panel-width, 18.571rem);
```

This is what we did for the slide panel component as well: https://github.com/cerner/terra-framework/blob/cca1b0c559589d62a967d99dab650c67a4a7a5af/packages/terra-slide-panel/src/SlidePanel.module.scss

Reply:
Fixed in https://github.com/cerner/terra-framework/pull/429/commits/82c3fd5b63ab11a574da1f8323fde4731425153b